### PR TITLE
[parametric] add remote config tests with empty `DD_SERVICE` or `DD_ENV`

### PR DESCRIPTION
**Pull request by lucaspimentel**
## Motivation

Some tracing libraries added checks for matching `service` and `env` in remote config payloads. For example, https://github.com/DataDog/dd-trace-java/pull/6636/. This causes issues in scenarios like SSI where the `env` (aka `DD_ENV`) is configured in the agent, but not on the tracer. The Java tracer is rejecting remote sampling rules in SSI, which prevents Adaptive Sampling from working correctly.

See also [APM_TRACING matching & SSI](https://docs.google.com/document/d/1lcrWRvd3n7Yxa-GiFq7GV1le2a43nfAvw5Iah1ACHzI/) 🔒

## Changes

Add a new test that asserts that the tracing library accepts a remote config payload without checking for matching `service` or `env`, since that is already handled by the agent.

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

**Review by cbeauchesne — APPROVED** [review]

File: manifests/php.yml [hunk 1]
```diff
@@ -331,6 +331,7 @@ tests/:
       TestDynamicConfigSamplingRules: missing_feature
       TestDynamicConfigTracingEnabled: missing_feature
       TestDynamicConfigV1: missing_feature
+      TestDynamicConfigV1_EmptyServiceTargets: v1.5.1 # version unknown
```
Comment (bwoebi) [inline manifests/php.yml:334]: 1.4.0 should be right.

Comment (lucaspimentel) [reply manifests/php.yml:334]:
```suggestion
      TestDynamicConfigV1_EmptyServiceTargets: v1.4.0
```

Comment (lucaspimentel) [reply manifests/php.yml]: Fixed, thanks!
